plugin/trino-bigquery/pom.xml

```diff
@@ -595,7 +589,7 @@
 
         <!-- Separate profile for Arrow related tests since they require extra JVM args -->
         <profile>
-            <id>cloud-tests-arrow-and-fte</id>
+            <id>cloud-tests-smoke</id>
```

Comment:
the name is misleading. It's smoke + "Important" (type-mapping, cleaner, metadata etc.)

Reply:
Renamed to cloud-tests-1 & cloud-tests-2.